App.css

```diff
@@ -0,0 +1,46 @@
+footer {
```

Comment:
You had to style the components based on the theme we had setup as per design system